libs/cart/src/guards/shipping-address/shipping-address.guard.ts

```diff
@@ -1,14 +1,15 @@
 import { CanActivate, Router } from '@angular/router';
 import { Observable } from 'rxjs';
 import { Injectable, Inject } from '@angular/core';
-import { tap } from 'rxjs/operators';
+import { tap, filter, switchMapTo, take } from 'rxjs/operators';
 
 import { DaffCartFacade } from '../../facades/cart/cart.facade';
 import { DaffCartShippingAddressGuardRedirectUrl } from './shipping-address-guard-redirect.token';
 
 /**
  * A routing guard that will redirect to a given url if the shipping address on the cart is not defined.
  * The url is `/` by default, but can be overridden with the DaffCartShippingAddressGuardRedirectUrl injection token.
+ * The guard will wait until the cart has been resolved before performing the check.
```

Comment:
Could you add "and emits" to the end of this doc? Just so it is clear that the observable won't emit until after the cart resolves.

Reply:
done